tests/mdns/test-multiple

```diff
@@ -39,11 +39,11 @@ main()
 
     if [[ "${OTBR_MDNS}" = 'mDNSResponder' ]]; then
         # dns-sd will not exit
-        dns_sd_check MultipleService1 _meshcop._udp 'nn=cool1 xp=1122334455667788'
-        dns_sd_check MultipleService2 _meshcop._udp 'nn=cool2 xp=1122334455667788'
+        dns_sd_check MultipleService1 _meshcop._udp 'nn=cool1 xp=ABCDEFGH'
```

Comment:
nit: is this a valid Extended PAN ID?

Reply:
Since we are now publishing the binary format of xpanid and the console output is printing it as character, the extpanid is actually '0x4142434445464748'.